apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -193,6 +193,11 @@ type OutputSpec struct {
 	//
 	// +optional
 	Secret *OutputSecretSpec `json:"secret,omitempty"`
+
+	// Fluentd including some plugins treats logs as a BINARY by default to forward.
+	// But a user sometimes wants to process logs depends on their requirements, e.g. handling char encoding correctly.
+	// +optional
+	CharEncoding string `json:"charEncoding,omitempty"`
```

Comment:
I believe this should 
* go on the "tuning" API instead of here and should be allowable for Elasticsearch and cloudwatch
* be enabled or disabled
* set the encoding via enablement instead of allowing admin to choose

@alanconway please advise how you wish to address in api

Reply:
I think we should not have any change in the API, and force this behavior for elasticsearch and cloudwatch with no admin changes.
This might turn out to be relevant for other cases and if it is we can make it tunable later. Let's not introduce new API till we are sure we have to.